app/components/Home/WalletLauncher.js

```diff
@@ -69,66 +131,143 @@ class WalletLauncher extends React.Component {
     }
   }
 
+  onSubmit = values => {
+    const { startLnd } = this.props
+    return startLnd(formToWalletFormat(values))
+  }
+
+  saveSettings = () => {
+    const { wallet, putWallet, showNotification, intl } = this.props
+    const formState = this.formApi.getState()
+    const { values } = formState
+
+    let updatedSettings = values
+
+    // if autopilot is off and no defaults are present, ensure they are included into config
+    // to properly fill in the form next time user decides to edit settings.
+    if (wallet.type === 'local' && !values.autopilot) {
+      updatedSettings = Object.assign({}, autopilotDefaults, values)
+    }
+
+    putWallet(formToWalletFormat(updatedSettings))
```

Comment:
With this, we will always be saving the default options onto the wallet config when the wallet form, is saved. This means that if the default settings are changed in the future (eg, when lnd increase the default max channel size) users that created wallets prior to this will not assume the new defaults as the old defaults will have been hardcoded into their wallet config.

Reply:
updated